### DISCUSSION TITLE
Product development

### DISCUSSION THREAD

**Discussion by Kslnicky** [lobehub/lobehub#2545] [Development Question] [answered]
Hi, I have a few questions Will there be a registration? Will there be a cloud version of database storage used, mongo db , superbase or something similar I noticed that when working with cyrillic for a long time, it starts to confuse cyrillic and latticine

**Comment by arvinxx** [comment, marked as answer]
Yes and stay tuned for it. https://github.com/lobehub/lobe-chat/discussions/1851
